apps/server/swagger/schemas.yml

```diff
@@ -414,27 +414,15 @@ components:
       required:
         - status
         - description
-        - batchErrors
-        - inProcessEntities
       type: object
       properties:
-        batchErrors:
-          type: array
-          description: List of all batch related errors, should be empty on success
-          items:
-            $ref: '#/components/schemas/BatchError'
         description:
           type: string
           description: Description of uploading Submission files
-        inProcessEntities:
-          type: array
-          description: Names of the entities that will be processed
-          items:
-            type: string
         status:
           type: string
           description: Result of uploading Submission files
-          enum: ['PROCESSING', 'INVALID_SUBMISSION', 'PARTIAL_SUBMISSION']
```

Comment:
how come PARTIAL is not required anymore? are we taking every submission as "complete"?